integrations/jenkins/client.py

```diff
@@ -45,6 +46,36 @@ async def get_builds(self) -> AsyncGenerator[list[dict[str, Any]], None]:
             event.attributes.setdefault(ResourceKey.BUILDS, []).extend(builds)
             yield builds
 
+    async def get_build_stages(self, build_url: str) -> list[dict[str, Any]]:
+        response = await self.client.get(f"{build_url}/wfapi/describe")
+        response.raise_for_status()
+        stages = response.json()["stages"]
+
+        # Enrich stages with full URLs
+        for stage in stages:
+            relative_url = stage["_links"]["self"]["href"]
+            stage["__fullUrl"] = f"{self.jenkins_base_url}{relative_url}"
```

Comment:
since we how have capability to use env.OCEAN_INTEGRATION_CONFIG_JENKINS_HOST, I think we can move this implementation to the config mapping and construct the full url there

Reply:
how so?

Reply:
Here's an example we did for Octopus Deploy to get the url

createMissingRelatedEntities: true
deleteDependentEntities: true
resources:
  - kind: space
    selector:
      query: "true"
    port:
      entity:
        mappings:
          identifier: .Id
          title: .Name
          blueprint: '"octopusSpace"'
          properties:
            url: env.OCEAN__INTEGRATION__CONFIG__SERVER_URL + "/app#/" + .Id
            description: .Description
            
            
            in your case, I see this mapping
            
           stageUrl: env.OCEAN__INTEGRATION__CONFIG__JENKINS_HOST  + ._links.self.href